src/main/java/net/fabricmc/installer/Handler.java

```diff
@@ -141,41 +156,31 @@ public void updateProgress(String text) {
 		statusLabel.setForeground(UIManager.getColor("Label.foreground"));
 	}
 
-	private void appendException(StringBuilder errorMessage, String prefix, String name, Throwable e) {
-		String prefixAppend = "  ";
-
-		errorMessage.append(prefix).append(name).append(": ").append(e.getLocalizedMessage()).append('\n');
-		for (StackTraceElement traceElement : e.getStackTrace()) {
-			errorMessage.append(prefix).append("- ").append(traceElement).append('\n');
-		}
-
-		if (e.getCause() != null) {
-			appendException(errorMessage, prefix + prefixAppend, Utils.BUNDLE.getString("prompt.exception.caused.by"), e.getCause());
-		}
-
-		for (Throwable ec : e.getSuppressed()) {
-			appendException(errorMessage, prefix + prefixAppend, Utils.BUNDLE.getString("prompt.exception.suppressed"), ec);
```

Comment:
Should these translation messages be removed in case these are their only usages?

Reply:
Probably, the Java printStackTrace doesn't bother with localizing them and it's somewhat counterproductive considering the stack trace being for devs

Reply:
I think I'll leave the lang string cleanup for later.